test/conformance/event/event_adapter_cuda.match

```diff
@@ -1,3 +1,4 @@
+urEventGetProfilingInfoTest.Success/NVIDIA_CUDA_BACKEND___{{.*}}_
```

Comment:
all the adapters apart from native cpu do implement at least some of these queries, why are all the tests failing?

Reply:
I just merged them in one test to save the timing and compare it at the end of queries. The main query that fail on all adapters is `UR_EVENT_STATUS_COMPLETE`

Reply:
I'd rather we were precise about which tests are failing, this could hide a future regression in a query that currently works

Reply:
Yeah, that's a good point. I have split that to 2 tests, one concerned with testing timings, and another for testing the success for the entry-point